balancers/config_test.go

```diff
@@ -9,159 +9,149 @@ import (
 	"github.com/ydb-platform/ydb-go-sdk/v3/internal/conn"
 )
 
-func TestFromConfig(t *testing.T) {
```

Comment:
Here and in other _test files

No need to spit test functions.
The code no need to be reused. Better to disable linter to the function or file with nolint directive:
https://golangci-lint.run/usage/false-positives/#nolint-directive

or by rule:
https://golangci-lint.run/usage/false-positives/#exclude-issues-by-path